Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabSaveLoadHandler.cpp

```diff
@@ -150,6 +139,17 @@ namespace AzToolsFramework
             return QDialogButtonBox::DestructiveRole;
         }
 
+        void PrefabSaveHandler::ExecuteSavePrefabDialog(const AZ::EntityId& entityId, bool useSaveAllPrefabsPreference)
```

Comment:
Will remove const ref according to [this](https://github.com/o3de/sig-core/blob/main/governance/C%2B%2B-Best-Practices-Guide.md#passing-parameters-by-value-vs-const-ref).

> As usual, consistency is recommended. For example, most existing O3DE APIs pass AZ::EntityId by value so it recommended that you do the same unless there is some special reason not to.